packages/mantine/src/components/date-range-picker/__tests__/DateRangePickerPopoverCalendar.spec.tsx

```diff
@@ -67,11 +70,12 @@ describe('DateRangePickerPopoverCalendar', () => {
         await user.click(screen.getAllByRole('button', {name: '14'})[0]);
 
         // hides the calendar when the second date is clicked
-        expect(screen.queryByRole('button', {name: '8'})).not.toBeInTheDocument();
+        expect(screen.queryAllByRole('button', {name: '8'})[0]).not.toBeVisible();
+        expect(screen.queryAllByRole('button', {name: '8'})[1]).not.toBeVisible();
```

Comment:
They are in the document?

Reply:
yes.. Only not visible (as seen from the logTestingPlaygroundURL() method)